src/Scraper/Application/Configurator.php

```diff
@@ -134,4 +140,10 @@ private function checkConfiguration($data, Collection $finalConfig)
             throw new ConfigurationException("Field(s) \"{$fieldsMissing}\" not found.", 0);
         }
     }
+
+    private function saveNewVariant($scrapedData, $variant): void
+    {
+        $scrapedData['variant'] = getVariantId($scrapedData['type'], $variant);
```

Comment:
Why is `getVariantId` an helper function? It expects a known data structure `variant` but it's an helper function. Wouldn't it deserve its own class?

Reply:
It is just a behaviour, it can be a helper or it can be a trait, I don't think it should be a class.